plugins/jobs/server/models/job.py

```diff
@@ -52,19 +56,33 @@ def _validateStatus(self, status):
             raise ValidationException(
                 'Invalid job status %s.' % status, field='status')
 
-    def list(self, user=None, limit=0, offset=0, sort=None, currentUser=None):
+    def list(self, user, types=None, statuses=None,
```

Comment:
We talked about this outside of the PR.  It is confusing, the best path forward while still maintaining backwards compatibility seemed to be a slightly awkward implementation sprinkled liberally with explanatory comments.